runners/flink/job-server/flink_job_server.gradle

```diff
@@ -82,10 +83,10 @@ dependencies {
   validatesPortableRunner project(path: ":beam-runners-core-java", configuration: "shadowTest")
   validatesPortableRunner project(path: ":beam-runners-reference-java", configuration: "shadowTest")
   compile project(path: ":beam-sdks-java-extensions-google-cloud-platform-core", configuration: "shadow")
-  compile library.java.slf4j_simple
 //  TODO: Enable AWS and HDFS file system.
   // For resolving external transform requests
   compile project(path: ":beam-sdks-java-io-kafka", configuration: "shadow")
+  shadowTestRuntimeClasspath library.java.slf4j_simple
```

Comment:
The Flink job server is meant to be a standalone application and not a library which is why we include the slf4j backend as part of it.

Reply:
done